src/adapters/beachfront.js

```diff
@@ -14,7 +14,7 @@ function BeachfrontAdapter() {
     const bids = bidRequests.bids || [];
     bids.forEach(function(bid) {
       var bidRequest = getBidRequest(bid);
-      var RTBDataParams = prepareAndSaveRTBRequestParams(bid);
+      var RTBDataParams = prepareAndSaveRTBRequestParams(bidRequest);
```

Comment:
ok but doesn't it just return here because `bid.params` doesn't exist?
https://github.com/bfmdeploy/Prebid.js/blob/420d921c0db56f06f87bb5004d3ba8dc3f1e4b13/src/adapters/beachfront.js#L46